packages/schematics/angular/service-worker/index.ts

```diff
@@ -171,13 +171,17 @@ export default function (options: ServiceWorkerOptions): Rule {
       throw new SchematicsException(`Service worker requires a project type of "application".`);
     }
 
+    const projectRoot = ('root' in project) ? project.root : '';
```

Comment:
This can be simplified to 
```
    const relativePathToWorkspaceRoot = project.root
      ? project.root.split('/').filter(x => x !== '').map(x => '..').join('/')
      : '';
```